packages/adapters/bitcoin/src/connectors/OKXConnector.ts

```diff
@@ -146,9 +164,19 @@ export class OKXConnector extends ProviderEventEmitter implements BitcoinConnect
       return undefined
     }
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let wallet: any = undefined
+
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const okxwallet = (window as any)?.okxwallet
-    const wallet = okxwallet?.bitcoin
+
+    // Bitcoin Testnet
+    if (params.requestedCaipNetworkId === TESTNET_CAIP_NETWORK_ID) {
+      wallet = okxwallet?.bitcoinTestnet
+    } else {
+      wallet = okxwallet?.bitcoin
```

Comment:
I think it would be better to have something like 
```
{
  [bitcoin.caipNetworkId]: 'bitcoin',
  [bitcoinTestnet.caipNetworkId]: 'bitcoinTestnet',
}
```
as it's easier to add other networks (eg devnet)